tox.ini

```diff
@@ -0,0 +1,6 @@
+[tox]
+skipsdist = true
+envlist = py27,py34,py35,py36,py37,py38,pypy2,pypy3,docs 
+
+[testenv]
+commands = python --version
```

Comment:
I did actually mean `--version` twice -- that way we can see the more specific version information